README.md

```diff
@@ -66,7 +66,7 @@ For installing a development version, see [here](devtools/install_dev.md).
 Once installed, a simple invocation of the `buildH` command will run the program (`$` represents the Unix prompt):
 
 ```
-$ buildH
+buildH
```

Comment:
Sorry to be picky. I would leave the `$` in since here it allows to see what we type and what is the output. Do you agree?